pkg/model/deployment.proto

```diff
@@ -77,6 +77,7 @@ message Deployment {
     // e.g. Update image from v1.5.0 to v1.6.0.
     string summary = 22;
     string version = 23;
```

Comment:
not really related to this PR but please add and TODO to remind remove this version from the deployment object later
> TODO: Remove version from deployment model.